paper-checkbox.html

```diff
@@ -171,6 +172,7 @@
         vertical-align: middle;
         padding-left: var(--paper-checkbox-label-spacing, 8px);
         white-space: normal;
+        line-height: normal;
```

Comment:
http://jsbin.com/ruveye/edit?html,output Seems ok; line-height is relative to font-size and [the spec](https://www.w3.org/TR/CSS2/visudet.html#propdef-line-height) kind of implies that `normal` is a constant. Also, line-height is apparently a minimum and contents of the line can expand this, so `normal` should cause the checkbox to expand to show the text regardless of what line-height the user sets on the host. Do you think this should have an actual test?